warehouse/integrations/postgres/testdata/docker-compose.postgres.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.9"
+
+services:
+  postgres:
+    image: postgres:15-alpine
```

Comment:
@lvrach shouldn't we mention in [our docs](https://www.rudderstack.com/docs/destinations/warehouse-destinations/postgresql/) a minimum version that we're compatible with? :thinking:

Reply:
Good point. Currently, I can't find anything in [docs](https://www.rudderstack.com/docs/resources/rudderstack-architecture/#database) or [Readme](https://github.com/rudderlabs/rudder-server/blob/master/README.md) mentioning it.

Reply:
what's the minimum version we support? 

For Postgres as rudder-server dependency, we already have a check https://github.com/rudderlabs/rudder-server/blob/a6f205db9f87fc02f5c5290ec30bfacd216f59bf/services/validators/envValidator.go#L193 it is 10 which make sense since is no longer support by postgres https://endoflife.date/postgresql

I believe we should have 11 as our oldest supported version and also check this during validation

Reply:
@lvrach Are you referring to destination validation here? During the validation for `Postgres` destination, we also check the version.

Reply:
We are currently not sure what min postgres we support as a destination.

Reply:
> We are currently not sure what min postgres we support as a destination.

:point_up: 

Also, might make sense to have our Warehouse integration test run twice, one for the minimum supported version and one for the latest.

Reply:
Corresponding task. https://linear.app/rudderstack/issue/PIPE-378/check-minimum-version-during-warehouse-validations

Reply:
Also, should we restrict this during the loading process as well? Maybe we can have a preliminary check with the existing customers, If everybody passes, we can enforce it else we need some action there.